app/src/main/java/com/github/se/signify/ui/screens/home/HomeScreen.kt

```diff
@@ -83,9 +77,9 @@ data class Exercise(val name: String, val route: String = "UNKNOWN_EXERCISE")
 fun HomeScreen(navigationActions: NavigationActions) {
   val defaultExercises =
       listOf(
-          Exercise("Easy", Screen.EXERCISE_EASY),
-          Exercise("Medium", Screen.EXERCISE_MEDIUM),
-          Exercise("Hard", Screen.EXERCISE_HARD))
+          ExerciseInformation(ExerciseLevel.Easy),
+          ExerciseInformation(ExerciseLevel.Medium),
+          ExerciseInformation(ExerciseLevel.Hard))
```

Comment:
## Requested changes

This should be computed using `ExerciseLevel.values()` to make space for future changes.

Reply:
I will take into account this change, but rather use `ExerciseLevel.entries`.